### ISSUE TITLE
convert Panel subclasses to ES6 class

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tasks#1049]
In https://github.com/phetsims/tasks/issues/1044#issuecomment-702358435 (conversion of common code to ES6 class), I said:
> `Panel` is going to be a lot of work, because there are 47 occurrences of "inherit( Panel," that must be converted first.

I've addressed all but 3 of those.  The remaining ones are a bit more work, and I'd like the responsible developer (@jbphet) to address them.  They include:

- [x] area-builder: FeedbackWindow and its 2 subclasses (@jbphet)
- [x] gene-expression-essentials: BiomoleculeToolboxNode (@jbphet)
- [x] projectile-motion: LabProjectileControlPanel (@jbphet)

@ariel-phet please prioritize. If possible, I'd like this to be completed when I return from vacation, so that I can proceed with converting `Panel` to ES6 class.

**Comment by pixelzoom** [comment]
I took care of neuron in MembranePotentialChart in https://github.com/phetsims/neuron/issues/145.

**Comment by pixelzoom** [comment]
Note that @jessegreenberg was originally assigned to this, but I converted Panels in his sims, and removed them from the list in https://github.com/phetsims/tasks/issues/1049#issue-713335871.

**Comment by pixelzoom** [comment]
@jbphet I converted (and tested) area-builder FeedbackWindow and its 2 subclasses in the above 3 commits. Please review.

**Comment by ariel-phet** [comment]
Sounds like @jbphet can fit this work in this week as it should be a quick review.

**Comment by pixelzoom** [comment]
@ariel-phet to clarify...   The review part of this issue will be quick.  Converting the remaining 2 classes listed in  https://github.com/phetsims/tasks/issues/1049#issue-713335871 will not be quick.

**Comment by jbphet** [comment]
The changes for area-builder's `FeedbackWindow` look great, and I tested the behavior and it appears correct.  I'll work on converting the two remaining panels listed above next.

**Comment by jbphet** [comment]
I've converted the usages in Gene Expression and Projectile Motion.  In both cases, I decided to make the panels subclasses of `Node` rather than `Panel` so that `super` could be called at the beginning of the constructor, thus allowing all the `this` usages, and then adding the `Panel` as a child at the end of the constructor.  The kept the code changes to a minimum and was more expedient than a more elaborate refactoring.

While testing the changes, I noticed that the layout of the control panel in Projectile Motion (the "Lab" screen) was a bit different from the currently published version.  I stashed my changes and re-tested, and the differences were still there, so this is not a result of the changes make for this issue.  I'll check in with the lead designer as to whether this matters and will log an issue if so.

@pixelzoom - back to you for review and any next steps.
